modules/ossm-about-smcp.adoc

```diff
@@ -21,3 +21,7 @@ Red Hat OpenShift Service on AWS (ROSA) places additional restrictions on where
 ====
 The {SMProductShortName} documentation uses `istio-system` as the example project, but you can deploy the service mesh to any project.
 ====
+
+ifndef::openshift-enterprise[]
+If you are deploying the control plane for use on {product-rosa}, see the Red Hat Knowledgebase article link:https://access.redhat.com/solutions/6529231[OpenShift service mesh operator istio basic not starting due to authentication errors], which discusses adding a new project and starting pods.
```

Comment:
```suggestion
If you are deploying the control plane for use on {product-rosa}, see the Red Hat Knowledgebase article link:https://access.redhat.com/solutions/6529231[OpenShift service mesh operator Istio basic not starting due to authentication errors], which discusses adding a new project and starting pods.
```
- Should Istio be capitalized? (even though it is not in the actual article title)